containerd-shim-v2/service.go

```diff
@@ -274,7 +329,23 @@ func getTopic(e interface{}) string {
 	return cdruntime.TaskUnknownTopic
 }
 
+func trace(ctx context.Context, tracer opentracing.Tracer, name string) opentracing.Span {
+	if tracer != opentracing.Tracer(nil) {
+		span, _ := opentracing.StartSpanFromContext(ctx, name)
+		span.SetTag("subsystem", "runtime")
```

Comment:
I think you could set this tag in the root span to avoid having to set every time here.

Reply:
@jodh-intel I tried setting it only at `New` where the tracer is created and don't see a subsystem for the other functions, so I'm leaving it in for now.